src/stores/settings/SettingsStore.tsx

```diff
@@ -35,28 +34,15 @@ export const useSettingsStore = createWithEqualityFn(
           return updatedState;
         }),
 
-      // Wallet
-      onWalletConnect: (activeWalletName: WalletConnected) => {
-        set({
-          activeWalletName,
-        });
-      },
-
-      onWalletDisconnect: () => {
-        set({
-          activeWalletName: 'none',
-        });
-      },
-
```

Comment:
Same here, although we did set the wallet name once the wallet connection started + cleared the name once the wallet disconnects - it´s never been used.

So, those to handlers got removed as well.